src/apps/copilots/src/pages/copilot-opportunity-details/tabs/copilot-applications/CopilotApplications.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, useMemo } from 'react'
+
+import { Table, TableColumn } from '~/libs/ui'
+
+import { CopilotApplication } from '../../../../models/CopilotApplication'
+import { FormattedMembers } from '../../../../services/members'
+
+import styles from './styles.module.scss'
+
+const tableColumns: TableColumn<CopilotApplication>[] = [
+    {
+        label: 'Topcoder Handle',
```

Comment:
Please make sure handle is link in new tab to member profile. For instance: `https://profiles.topcoder-dev.com/kirildev` would be for `kirildev` handle.